UTD/CMakeLists.txt

```diff
@@ -27,7 +27,12 @@ target_sources(timer PRIVATE "src/utils/timer.cpp")
 ### String
 add_library(string SHARED)
 set_target_properties(string PROPERTIES CXX_STANDARD 20)
-target_sources(string PRIVATE "src/data-structures/string32.cpp")
+target_sources(string PRIVATE "src/data-structures/string.cpp")
+
+### String
+add_library(string32 SHARED)
```

Comment:
Should we just have a single string library instead for simplicity?

Reply:
think thats a good suggestion, let me change it